Makefile

```diff
@@ -47,7 +47,7 @@ HELP_FUN = \
 		   }
 
 statusgo: ##@build Build status-go as statusd server
-	go build -i -o $(GOBIN)/statusd -v -tags '$(BUILD_TAGS)' $(shell _assets/build/testnet-flags.sh) ./cmd/statusd
```

Comment:
We may leave this one but remove `-X github.com/status-im/status-go/geth/params.UseMainnetFlag=false` part.

Reply:
As we don't separate between mainnet and testnet compiling anymore I would prefer to set the optional flags as Makefile variable.

Reply:
Ok, that makes sense.